.travis.yml

```diff
@@ -65,13 +65,15 @@ jobs:
             - liblua5.3-dev
             - lua5.3
     # Note: luarocks does not work with Lua 5.0.
-    - env: LUA=5.1 LUANAME=lua5.1 BUILD_IN_DIR=/tmp/awesome-build
+    - env: LUA=5.1 LUANAME=lua5.1 BUILD_IN_DIR=/tmp/awesome-build WITH_XCB_ERRORS=yes
```

Comment:
Sorry for noticing earlier, but it would be better to add this to the `DO_COVERAGE=codecov` job.

Reply:
Went ahead and amended your last commit.